pkg/types/aws/validation/machinepool.go

```diff
@@ -7,9 +7,26 @@ import (
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/apimachinery/pkg/util/validation/field"
 
+	"github.com/openshift/installer/pkg/types"
 	"github.com/openshift/installer/pkg/types/aws"
 )
 
+var (
+	validArchitecturesAWS = map[types.Architecture]bool{
+		types.ArchitectureAMD64: true,
+		types.ArchitectureARM64: true,
+	}
+
+	// ValidArchitectureValuesAWS lists the supported arches for AWS
+	ValidArchitectureValuesAWS = func() []string {
```

Comment:
The AWS in these fields is redundant with the package name.
```suggestion
	ValidArchitectureValues = func() []string {
```

Reply:
Renamed and squashed.